uaclient/entitlements/cis.py

```diff
@@ -36,9 +36,9 @@ def messaging(self,) -> Dict[str, List[Union[str, Tuple[Callable, Dict]]]]:
         return messages
 
     @property
-    def packages(self) -> List[str]:
+    def packages(self) -> Optional[List[str]]:
         if self._called_name == "usg":
-            return []
+            return None
```

Comment:
This change was made so that in `fips.py` we can return an empty list for `packages` but `enable` will still call `install_packages`.

Now `install_packages` is called even for len 0 `packages`, but is not called if `packages` is None

Reply:
I think I am missing something, looking at the modification on `install_packages`, wouldn't it work with the empty list as well ?

Reply:
The problem is that `install_packages` won't even get called with an empty list. I left a longer comment on this below